{{cookiecutter.project_slug}}/clients/vue3/tailwind.config.js

```diff
@@ -0,0 +1,28 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ['./index.html', './src/**/*.{vue,js}'],
+  theme: {
+    extend: {
+      colors: {
+        primary: '#042642',
+        accent: '#d93a00',
+        success: '#4faf64',
+        warning: '#f4b942',
+        error: '#d72638',
+      },
+      fontFamily: {
+        avenir: ['Avenir', 'Helvetica', 'Arial', 'sans-serif'],
+      },
+    },
+    container: {
+      padding: {
+        DEFAULT: '1rem',
+        sm: '2rem',
+        lg: '4rem',
+        xl: '5rem',
+        '2xl': '6rem',
+      },
+    },
```

Comment:
**Info:** Add the `container` object to the Tailwind config for if we want to be able to specify different padding amounts for each breakpoint. Can provide a default value and any breakpoint-specific overrides.

Reference:
https://tailwindcss.com/docs/container